### DISCUSSION TITLE
What does "nn" mean in nnUNet?

### DISCUSSION THREAD

**Discussion by notmatthancock** [MIC-DKFZ/nnUNet#728] [Q&A] [answered]
Sorry for the silly question, but I didn't see this in the README or the Nature paper. Is "nn" an acronym for something? Is it significant?

**Comment by FabianIsensee** [comment, marked as answer]
nnU-Net = 'no new net'. In constrast to most other segmentation papers we do not use fancy architectural modifications. We do not believe that this is necessary for the current landscape of segmentation datasets
